### PR TITLE
phase 2 blog

### PR DESCRIPTION
**Pull request by leanthebean**
## Summary
phase 2 blog

## Testing Plan
local

## Breaking Change
No

### CODE REVIEW

File: blog/2022-05-10-phase2.mdx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  Fish Incentivized Testnet!
+</Highlight>
+
+## What happened with Phase 1?
+
+Phase 1 ran from December 1st 2021 to March 11th 2022. At the end, we’ve had over 10k active participants, and almost 11k unique miners. Our goal was for Iron Fish to have the biggest L1 incentivized testnet by participation, and it looks like we succeeded.
+
+The [Phase 1 leaderboard](https://phase1.testnet.ironfish.network/leaderboard) and [explorer](https://phase1.explorer.ironfish.network) are saved for historic/archival reasons.
+
+Phase 1 token distribution will happen at the same time as Phase 2 token distribution and closer to mainnet. When mainnet? When all the engineering tasks on our [roadmap](https://ironfish.network/roadmap) are complete. If you want this to go faster, consider applying on our [careers page](https://ironfish.network/careers/) for a full time engineering job at Iron FIsh :) or help us with open source contribution as part of the incentivized testnet program.
```
Comment (brekk) [inline blog/2022-05-10-phase2.mdx:39]: `s/FIsh/Fish/g`

File: blog/2022-05-10-phase2.mdx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+What’s new in Phase 2?
```
Comment (brekk) [inline blog/2022-05-10-phase2.mdx]: Should this be a header?
```suggestion
## What’s new in Phase 2?
```

File: blog/2022-05-10-phase2.mdx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+Phase 2 is meant to stress-test Iron Fish as well as incorporate some of the feedback we’ve heard from the community about Phase 1. We heard your feedback, and we’re making some changes!
+
+Phase 2 will have 4 categories (described in even more detail on the [testnet site](https://testnet.ironfish.network/about)):
+
+### Hosting a node
+
+Running an Iron Fish full node with telemetry on and a registered graffiti set gets you max of 10 points for every 12 consecutive hours of hosting (even if running multiple nodes). Points will be automatically assigned to you on the leaderboard. To make sure your node is properly set up to earn points, we made a custom command `ironfish testnet`. Go ahead and run that command before starting your node to make sure you’re all set.
```
Comment (deekerno) [inline blog/2022-05-10-phase2.mdx:49]: Consider "...and setup with a registered graffiti".

File: blog/2022-05-10-phase2.mdx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+### Reporting a bug
+
+If you see a bug, report it on our github issues page and submit the form to get points! One quality bug found is 100 points.
+
+### Open Source Contributions (Submitting a PR)
+
+Iron Fish is in heavy development. Not only are we focused on the bigger roadmap items for engineerings, but we’re constantly working on improving user experience. Contributing to the ironfish repo is immensely appreciated as it directly impacts the project getting to mainnet faster.
+
+We appreciate open source contributions so much that we are allocating a <b>_separate pool of coins_</b> just for PR contributions. Open source developers no longer have to compete with miners for points in the same pool.
```
Comment (brekk) [inline blog/2022-05-10-phase2.mdx:65]: Minor but I think you can do `*_separate pool of coins_*` to get italic + bold

File: blog/2022-05-10-phase2.mdx [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+Note that while we’ve removed earning points for mining a block from Phase 2, the only way to get coins is by mining them. It is now possible (and easy!) to create and join pools. Feel free to [join the Iron Fish team pool](https://ironfish.network/docs/onboarding/miner-iron-fish#join-a-mining-pool) if you can’t find others. The faucet is disabled for this phase.
+
+### Reporting a bug
+
+If you see a bug, report it on our github issues page and submit the form to get points! One quality bug found is 100 points.
+
+### Open Source Contributions (Submitting a PR)
+
+Iron Fish is in heavy development. Not only are we focused on the bigger roadmap items for engineerings, but we’re constantly working on improving user experience. Contributing to the ironfish repo is immensely appreciated as it directly impacts the project getting to mainnet faster.
```
Comment (deekerno) [inline blog/2022-05-10-phase2.mdx:63]: Linking to the repository may be good.

File: blog/2022-05-10-phase2.mdx [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+      backgroundColor: color,
+      borderRadius: "2px",
+      padding: "0.2rem",
+    }}
+  >
+    {children}
+  </span>
+);
+
+####
```
Comment (brekk) [inline blog/2022-05-10-phase2.mdx:26]: Is this a header?

File: blog/2022-05-10-phase2.mdx [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+<Highlight color="#FFEC1F">
+  On Tuesday, May 10th 2022 at 12pm PST, we’re launching Phase 2 of the Iron
+  Fish Incentivized Testnet!
+</Highlight>
+
+## What happened with Phase 1?
+
+Phase 1 ran from December 1st 2021 to March 11th 2022. At the end, we’ve had over 10k active participants, and almost 11k unique miners. Our goal was for Iron Fish to have the biggest L1 incentivized testnet by participation, and it looks like we succeeded.
+
+The [Phase 1 leaderboard](https://phase1.testnet.ironfish.network/leaderboard) and [explorer](https://phase1.explorer.ironfish.network) are saved for historic/archival reasons.
```
Comment (brekk) [inline blog/2022-05-10-phase2.mdx:37]: Minor, but probably adding a space between the slash here will help page flow

**Review by deekerno — APPROVED** [review]

File: blog/2022-05-10-phase2.mdx (hunk 6 above)
Comment (leanthebean) [reply blog/2022-05-10-phase2.mdx:26]: fixed

File: blog/2022-05-10-phase2.mdx (hunk 4 above)
Comment (leanthebean) [reply blog/2022-05-10-phase2.mdx:65]: fixed

File: blog/2022-05-10-phase2.mdx (hunk 2 above)
Comment (leanthebean) [reply blog/2022-05-10-phase2.mdx]: fixed

File: blog/2022-05-10-phase2.mdx (hunk 5 above)
Comment (leanthebean) [reply blog/2022-05-10-phase2.mdx]: fixed

**Pull request merged**